src/controllers/signup.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi');
+const {
+  addUser,
+} = require('../database/queries/addData');
+const hashPassword = require('../helpers/hashPass');
+const {
+  signUpSchema,
+} = require('./../helpers/validation-shemas');
+
+exports.get = (req, res) => {
+  res.render('signup', {
+    layout: 'login_signup',
+    title: 'signup ',
+    js: ['collectData', 'signup'],
+    css: ['signup'],
+  });
+};
+
+exports.post = (req, res) => {
+  const user = req.body;
+  const UserInfo = {
+    ...user.firstSection,
+    ...user.secondSection,
+    ...user.thirdSection,
+  };
+  console.log(UserInfo.password);
+  const {
+    error,
+  } = Joi.validate(UserInfo, signUpSchema);
+  if (!error) {
+    hashPassword(UserInfo.password).then((hashedPass) => {
+      UserInfo.password = hashedPass;
+      addUser(UserInfo)
+        .then(() => {
+          res.status(201).send(JSON.stringify({
+            success: 'Regestration success ...',
+            user: UserInfo,
```

Comment:
@Fatmasiam For sake of security, I think you should remove `password` from the userInfo that you send back to the user

Reply:
@tomduggan85  
thanx for your warning me on this point, 
I forget it.